libraries/linters/snippet_comparison.py

```diff
@@ -0,0 +1,264 @@
+'''
+   snippet_comparison.py
+'''
+
+import re
+import httplib2
+
+class snippet_comparison(object):
+	def __init__(self,book,chap,chnk):
+		self.book = book
+		self.chap = chap
+		self.chnk = chnk
+		self.httplib2_instance = httplib2.Http()
+		
+		self.book_directory = {
```

Comment:
The books and their codes are already in this library: https://github.com/unfoldingWord-dev/tx-manager/blob/develop/libraries/door43_tools/bible_books.py .